content/en/02-installing-pixie/04-install-schemes/01-cli.md

```diff
@@ -103,22 +103,21 @@ rpm -i pixie-px.x86_64.rpm
   Please refer to <a href="/reference/admin/environment-configs">Environment-Specific Configurations</a> for other configurations that should be set for your specific Kubernetes environment.
 </Alert>
 
-```bash
+<TemplatedCodeBlock
+  code={`
 # List Pixie deployment options.
-px deploy --help
-
+px deploy --help \n
 # Deploy the Pixie Platform in your K8s cluster (No OLM present on cluster).
```

Comment:
@aimichelle leaving the empty lines in this template string caused the following compilation issues:
```
 ERROR #11321  PLUGIN

"gatsby-plugin-mdx" threw an error while running the onCreateNode lifecycle:

unknown: Unexpected token, expected "{" (1:7)

> 1 | export PX_CLOUD_ADDR=@PLACEHOLDER@
    |        ^
  2 | px deploy --extract_yaml ./ --deploy_key <PIXIE_DEPLOYMENT_KEY> --deploy_olm=false \n
  3 | # Extract YAML (Self-hosting Pixie Cloud).
  4 | px deploy --extract_yaml ./ --deploy_key <PIXIE_DEPLOYMENT_KEY> --dev_cloud_namespace plc \n/home/ddelnano/code/docs.px.dev/content/en/02-installing-pixie/04-install-schemes/02-yaml.md: unknown: Unexpected token, expected "{" (1:7)

> 1 | export PX_CLOUD_ADDR=@PLACEHOLDER@
    |        ^
  2 | px deploy --extract_yaml ./ --deploy_key <PIXIE_DEPLOYMENT_KEY> --deploy_olm=false \n
  3 | # Extract YAML (Self-hosting Pixie Cloud).
  4 | px deploy --extract_yaml ./ --deploy_key <PIXIE_DEPLOYMENT_KEY> --dev_cloud_namespace plc \n

  31 |   const createAndProcessNode = path => {
  32 |     const fileNodePromise = createFileNode(path, createNodeId, pluginOptions).then(fileNode => {
> 33 |       createNode(fileNode);
     |       ^
  34 |       return null;
  35 |     });
  36 |     return fileNodePromise;

File: node_modules/gatsby-source-filesystem/gatsby-node.js:33:7



  SyntaxError: unknown: Unexpected token, expected "{" (1:7)
  > 1 | export PX_CLOUD_ADDR=@  9mPLACEHOLDER@
      |        ^
    2 | px deploy --extract_yaml ./ --deploy_ke  y <PIXIE_DEPLOYMENT_KEY> --deploy_olm=false  [39m \n
    3 | # Extract YAML (Self-hosting   Pixie Cloud).
    4 | px deploy --extract_yaml ./ --deploy_ke  y <PIXIE_DEPLOYMENT_KEY> --dev_cloud_namespace plc   1m\n/home/ddelnano/code/docs.px.dev/content/en/02-installing-pixie/04-instal  l-schemes/02-yaml.md: unknown: Unexpected token, expected "{" (1:7)
  > 1 | export PX_CLOUD_ADDR=@  9mPLACEHOLDER@
      |        ^
    2 | px deploy --extract_yaml ./ --deploy_ke  y <PIXIE_DEPLOYMENT_KEY> --deploy_olm=false  [39m \n
    3 | # Extract YAML (Self-hosting   Pixie Cloud).
    4 | px deploy --extract_yaml ./ --deploy_ke  y <PIXIE_DEPLOYMENT_KEY> --dev_cloud_namespace plc   1m\n
  SyntaxError: unknown: Unexpected token, expected "{" (1:7)
  > 1 | export PX_CLOUD_ADDR=@  9mPLACEHOLDER@
      |        ^
    2 | px deploy --extract_yaml ./ --deploy_ke  y <PIXIE_DEPLOYMENT_KEY> --deploy_olm=false  [39m \n
    3 | # Extract YAML (Self-hosting   Pixie Cloud).
    4 | px deploy --extract_yaml ./ --deploy_ke  y <PIXIE_DEPLOYMENT_KEY> --dev_cloud_namespace plc   1m\n/home/ddelnano/code/docs.px.dev/content/en/02-installing-pixie/04-instal  l-schemes/02-yaml.md: unknown: Unexpected token, expected "{" (1:7)
  > 1 | export PX_CLOUD_ADDR=@  9mPLACEHOLDER@
      |        ^
    2 | px deploy --extract_yaml ./ --deploy_ke  y <PIXIE_DEPLOYMENT_KEY> --deploy_olm=false  [39m \n
    3 | # Extract YAML (Self-hosting   Pixie Cloud).
    4 | px deploy --extract_yaml ./ --deploy_ke  y <PIXIE_DEPLOYMENT_KEY> --dev_cloud_namespace plc   1m\n
```

Adding `\n` here was required to keep the line spacing the same. The other thing I tried was adding `\n` on its own line, but that introduces more spacing than the approach I took as seen in the highlighted text below

![Screen Shot 2024-07-11 at 2 40 02 PM](https://github.com/user-attachments/assets/7e287eac-e8b2-44c4-9526-ea99d108e418)

If you have any suggestions here, let me know.